tpl/template_funcs.go

```diff
@@ -1803,7 +1803,7 @@ func htmlUnescape(in interface{}) (string, error) {
 
 func init() {
 	funcMap = template.FuncMap{
-		"absURL":       func(a string) template.HTML { return template.HTML(helpers.AbsURL(a)) },
+		"absURL":       func(a interface{}) template.HTML { return template.HTML(helpers.AbsURL(cast.ToString(a))) },
```

Comment:
@moorereason please stick to this PR. Taking the entire commit history of Hugo into consideration for this simple change is ... just stupid.